main.go

```diff
@@ -228,3 +129,60 @@ func normalizeAddr(s string) string {
 
 	return u.String()
 }
+
+
+// parseConfig is broken into its own function for testability
```

Comment:
A standard function comment describing the purpose would preferable.

Reply:
Yes, good point. I suppose I'll just delete that comment as it's more aimed at explaining to reviewers (you haha) why I broke it out.